content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -224,6 +222,34 @@ It is recommended that you disable this plugin in the scheduling profile when
 using default constraints for `PodTopologySpread`.
 {{< /note >}}
 
+#### A kubernetes default
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+When the feature gate `DefaultPodTopologySpreadPlugin` is enabled, the following
+default constraints for `PodTopologySpreadPlugin` are applied:
+
+```yaml
+defaultConstraints:
+  - maxSkew: 3
+    topologyKey: "kubernetes.io/hostname"
+    whenUnsatisfiable: ScheduleAnyway
+  - maxSkew: 5
+    topologyKey: "topology.kubernetes.io/zone"
+    whenUnsatisfiable: ScheduleAnyway
+```
+
+Also, the legacy `SelectorSpread` plugin is disabled.
+
+{{< note >}}
+If your nodes are not expected to have **both** `kubernetes.io/hostname` and
+`topology.kubernetes.io/zone` we recommend that you define your own constraints
+instead of using the kubernetes default.
+
+The `PodTopologySpread` plugin ignores all nodes that don't have both topologies
+from scoring.
```

Comment:
Does this refer to the two topology keys `kubernetes.io/hostname` and `topology.kubernetes.io/zone`? I wasn't sure.

Reply:
Yes. But let me describe it more generally.

The `PodTopologySpread` plugin gives a positive score to nodes if and only if the node has all the topologies used in the spreading constraints. In the case of the default, this resolves to hostname and zone.